web/control_flow/using_template_repeat_with_a_table_row_or_an_option/my_element.html

```diff
@@ -1,4 +1,4 @@
-<polymer-element name="my-element">
```

Comment:
This tutorial probably needs a big update, as this isn't supported in the new polymer (yet at least). You can mimic it just by using regular divs and table styles though, something like the following:

```html
<dom-module id="my-element">
  <template>
    <style>
      .table {
        display: table;
      }
      .row {
        display: table-row;
      }
      .cell {
        display: table-cell;
      }
    </style>
    <div class="table">
      <template is="dom-repeat" items="{{fruits}}" as="fruit">
        <div class="row">
          <div class="cell">{{fruit}}</div>
        </div>
      </template>
    </div>
  </template>
</dom-module>
```

I don't know of a way to do this for `<option>` elements though, so this example would probably need to be renamed to remove that.